### DISCUSSION TITLE
Missing `ant` on macOS-latest

### DISCUSSION THREAD

**Discussion by ialarmedalien** [community/community#25359] [Actions]
<p>Attempting to test project builds on different OSes, the version running on macOS-latest fails with the message:</p>
<blockquote>
<p><code>ant -Djardir=submodules/jars/lib/jars/ -DKBASE_COMMON_JAR=kbase/common/kbase-common-0.0.23.jar</code><br>
<code>make: ant: No such file or directory</code><br>
<code>make: *** [compile] Error 1</code></p>
</blockquote>
<p>The workflow is extremely simple – checkout repo, set up JDK v1.8, and run the <code>make</code> command, which checks out some git submodules and then attempts to run ant. It does not look like <code>ant</code> is in the path for the macOS runner as <code>which ant</code> does not return a result.</p>
<p>See <a href="https://github.com/ialarmedalien/kb_sdk/runs/766492449?check_suite_focus=true" rel="nofollow noopener">https://github.com/ialarmedalien/kb_sdk/runs/766492449?check_suite_focus=true</a> for the workflow.</p>

**Comment by cschleiden** [comment, marked as answer]
<p>You are right, <code>ant</code> is not in the set of installed software for MacOS - see <a href="https://github.com/actions/virtual-environments/blob/d1fc6628feab4f809520367065199344535defde/images/macos/macos-10.15-Readme.md" rel="nofollow noopener">https://github.com/actions/virtual-environments/blob/d1fc6628feab4f809520367065199344535defde/images/macos/macos-10.15-Readme.md</a>.</p>
<p>You could create a request for the tool here: <a href="https://github.com/actions/virtual-environments/issues/new?assignees=&amp;labels=t%3AEnhancement&amp;template=tool-request.md&amp;title=Update%2FAdd+%5Btool+name%5D" rel="nofollow noopener">https://github.com/actions/virtual-environments/issues/new?assignees=&amp;labels=t%3AEnhancement&amp;template=tool-request.md&amp;title=Update%2FAdd+[tool+name]</a></p>
